roadside-forms-frontend/frontend_web_app/src/components/CommonForm/vehicleInfo.js

```diff
@@ -34,18 +34,19 @@ export const VehicleInfo = (props) => {
   }, [driversLicenceJurisdiction]);
 
   const fetchICBCVehicleInfo = () => {
-    // if (values["vehicle_plate_no"]) {
-    //   ICBCVehicleDataApi.get(values["vehicle_plate_no"]).then((resp) => {
-    //     if (!_.isEmpty(resp.data)) {
-    //       setFieldValue("driver_last_name", party.lastName);
-    //       setFieldValue("driver_given_name", party.firstName);
-    //       setFieldValue("driver_dob", new Date(party.birthDate));
-    //       setFieldValue("driver_address", address.addressLine1);
-    //       setFieldValue("driver_city", address.city);
-    //       setFieldValue("driver_postal", address.postalCode);
-    //     }
-    //   });
-    // }
+    if (values["vehicle_plate_no"]) {
```

Comment:
Can we remove the commented out code below? Also - let's try to keep console logs out of the main branch unless they're super necessary